tests/test_runs.py

```diff
@@ -87,7 +87,7 @@ def test_tv_run_inversion(persistent_temp_model, monkeypatch):
 
     if alpha_active:
 
-        fwd_alpha = mdl_out.solvers[0].forward_alpha
+        fwd_alpha = mdl_out.solvers[0].forward
```

Comment:
No `[0]` here

Reply:
@jrmaddison this part was like this from before... I just modify the forward_alpha to forward ... I don't know what the zero is for?

Reply:
Apologies, I misread